src/components/App.tsx

```diff
@@ -253,12 +253,10 @@ export type ExcalidrawImperativeAPI =
       >["getSceneElementsIncludingDeleted"];
       history: {
         clear: InstanceType<typeof App>["resetHistory"];
-        resumeRecording: InstanceType<typeof App>["resumeRecording"];
```

Comment:
removed from API in favor of adding a `commitToHistory` param to `updateScene` — I don't wanna expose low-level history API to host apps yet because history is gonna be rewritten at one point.